builder/image/Dockerfile

```diff
@@ -61,5 +52,7 @@ ADD etc /etc
 ADD . /app
 RUN chown -R root:root /app
 
+RUN /app/bin/install-buildpacks.sh
```

Comment:
I think it's is better to use the herokuish buildpack versions as those are tested. BP versions should be aligned soon:
https://github.com/gliderlabs/herokuish/pull/9
https://github.com/gliderlabs/herokuish/pull/10